src/test/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorServiceSpec.scala

```diff
@@ -144,7 +144,7 @@ class PolicyEvaluatorServiceSpec extends AnyFlatSpec with Matchers with TestSupp
   protected def clearDatabase(): IO[Unit] = IO(TestSupport.truncateAll).void
 
   private[service] def savePolicyMembers(policy: AccessPolicy) = {
-    policy.members.toList.parTraverse {
+    policy.members.toList.traverse {
```

Comment:
given the small database there are too many serializable failures with `parTraverse` so it turns out to be slower